packages/sd-jwt/package.json

```diff
@@ -0,0 +1,78 @@
+{
+  "name": "@veramo/sd-jwt",
+  "description": "Veramo plugin to enable SD-JWT",
+  "version": "5.6.0",
+  "main": "build/index.js",
+  "exports": {
+    ".": "./build/index.js"
+  },
+  "types": "build/index.d.ts",
+  "scripts": {
+    "build": "tsc",
+    "generate-plugin-schema": "node ../cli/bin/veramo.js dev generate-plugin-schema"
+  },
+  "veramo": {
+    "pluginInterfaces": {
+      "ISDJwtPlugin": "./src/types.ts"
+    }
+  },
+  "dependencies": {
+    "@sphereon/ssi-sdk-ext.did-utils": "^0.16.0",
+    "@sd-jwt/core": "0.3.2-next.107",
```

Comment:
do all of these sd-jwt packages support browser and react native (except for `crypto-nodejs`)?

Reply:
I'll check on them and let you know :)

Reply:
Yeah, It works regardless of platform except crypto-nodejs